.github/workflows/soundness.yml

```diff
@@ -200,3 +204,18 @@ jobs:
         run: |
           curl -s https://raw.githubusercontent.com/swiftlang/github-workflows/refs/heads/main/.github/workflows/configs/yamllint.yml > /tmp/yamllint.yml
           yamllint --strict --config-file /tmp/yamllint.yml ${GITHUB_WORKSPACE}
+  python-lint-check:
+    name: Python lint check
+    if: ${{ inputs.python_lint_check_enabled }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+      - name: Run flake8
+        run: |
+          pip3 install flake8 flake8-import-order --break-system-packages
```

Comment:
As you may know, it seems that `ubuntu-latest` on GitHub-hosted runners is currently in the process of migrating to Ubuntu 24.04. Until it's completed, Ubuntu 22.04 may be used, and on Ubuntu 22.04, this command will fail because its `pip3` is old and does not have `--break-system-packages`.

Announcement: https://github.com/actions/runner-images/issues/10636

Failure case: https://github.com/kkebo/swift-format/actions/runs/11299915833/job/31431847497

Since the migration will be completed soon, I think we can ignore it.

Reply:
Just fixed it: https://github.com/swiftlang/github-workflows/pull/28

Reply:
Thank you!